### PR TITLE
Create homepage, login

### PR DESCRIPTION
**Pull request by lerhhl**
Create a homepage, login and logout pages.

### CODE REVIEW

File: app/controllers/registrations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  end
+
+  def account_update_params
+    params.require(:user).permit(:firstname, :lastname, :email, :password, :password_confirmation, :current_password, :avatar, :avatar_cache, :remove_avatar)
+  end
+
+  protected
+    def after_update_path_for(resource)
+      edit_user_registration_path
+    end
```
Comment (gohdaniel15) [inline app/controllers/registrations_controller.rb]: These shouldn't be nested. They should be on the same indentation level as `protected` 😄

File: app/views/devise/registrations/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,72 @@
+<h2>Edit <%= resource_name.to_s.humanize %></h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
+  <%= devise_error_messages! %>
+
+
```
Comment (gohdaniel15) [inline app/views/devise/registrations/edit.html.erb]: extra blank line detected!

File: app/views/devise/registrations/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    <%= f.label :first_name %> <br />
+    <%= f.text_field :first_name %>
+  </div>
+  <div class="field">
+    <%= f.label :last_name %> <br />
+    <%= f.text_field :last_name %>
+  </div>
+
+
+
```
Comment (gohdaniel15) [inline app/views/devise/registrations/edit.html.erb]: extra blank line(s) detected!

File: app/views/devise/registrations/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+      <div class="actions">
+        <%= f.submit "Sign up" %>
+      </div>
+      <div class="field">
+      Already Registered?  <%= render "devise/shared/links" %>
+      </div>
+    <% end %>
+  </div>
+
+
```
Comment (gohdaniel15) [inline app/views/devise/registrations/new.html.erb]: extra blank line detected

File: app/views/layouts/_header.html.erb [hunk 5]
```diff
@@ -0,0 +1,19 @@
+<div class="container col-lg-8 col-lg-offset-2">
+<header>
+    <div class="">
+
```
Comment (gohdaniel15) [inline app/views/layouts/_header.html.erb]: extra blank line detected

File: app/views/layouts/_header.html.erb [hunk 6]
```diff
@@ -0,0 +1,19 @@
+<div class="container col-lg-8 col-lg-offset-2">
+<header>
+    <div class="">
+        
+        <nav>
+            <%= link_to "Twitter Clone", root_path, id: "logo" %>
+            <ul id='nav'>
+                <li><%= link_to "Home", root_path %></li>
+                <% if !user_signed_in? %>
+                    <li><%= link_to "Log in", new_user_session_path %></li>
```
Comment (gohdaniel15) [inline app/views/layouts/_header.html.erb]: Inconsistent quotation marks. Are we going with single quotes or double quotes? Pick one, and keep it consistent :)

**Review by gohdaniel15 — APPROVED** [review]
LGTM! But consider comments. We should keep our blank line spacing consistent. One blank line to separate our chunks of code is fine. 

Other than that, this looks good :) Good work on the formatting for the rest of the code! 🚀

File: app/controllers/registrations_controller.rb (hunk 1 above)
Comment (lerhhl) [reply app/controllers/registrations_controller.rb]: Thank you for the feedback.

File: app/views/devise/registrations/edit.html.erb (hunk 2 above)
Comment (lerhhl) [reply app/views/devise/registrations/edit.html.erb]: Thank you for the feedback.

File: app/views/devise/registrations/new.html.erb (hunk 4 above)
Comment (lerhhl) [reply app/views/devise/registrations/new.html.erb]: Thank you for the feedback.

**Pull request merged**